pwiz_tools/Skyline/Menus/ViewMenu.cs

```diff
@@ -301,6 +305,16 @@ private void zMenuItem_Click(object sender, EventArgs e)
             GraphSpectrumSettings.ShowZIons = !GraphSpectrumSettings.ShowZIons;
         }
 
+        private void zhMenuItem_Click(object sender, EventArgs e)
+        {
+            GraphSpectrumSettings.ShowZHIons = !GraphSpectrumSettings.ShowZHIons;
```

Comment:
These are really old handlers you are copying. Please change them all to call public functions like:
ShowAIons(), ShowBIons(), ... ShowZhhIons()

Then you (and anyone else) can call them from a test rather than changing Settins.Default which does not update the UI.